lib/descriptors/ProtoDescriptor.js

```diff
@@ -50,23 +48,39 @@ ProtoDescriptor.prototype.inspect = function () {
 
 /** @override */
 ProtoDescriptor.prototype.toTemplateObject = function () {
-  if (this._recursing) {
-    throw new Error('import loop detected: ' + this.getName())
-  }
-  this._recursing = true
```

Comment:
ya, exactly. now that we evaluate it lazily, it's fine if there are infinite loops.